core/src/main/scala/org/genivi/sota/core/DeviceDirectives.scala

```diff
@@ -0,0 +1,48 @@
+/**
+  * Copyright: Copyright (C) 2016, ATS Advanced Telematic Systems GmbH
+  * License: MPL-2.0
+  */
+
+package org.genivi.sota.data
+
+import akka.http.scaladsl.server.{AuthorizationFailedRejection, Directive1, Directives}
+import eu.timepit.refined.api.Refined
+import org.genivi.sota.common.DeviceRegistry
+import org.genivi.sota.http.AuthedNamespaceScope
+import org.genivi.sota.device_registry.common.{Errors => DeviceRegistryErrors}
+import akka.http.scaladsl.marshalling.Marshaller._
+import org.genivi.sota.http.UuidDirectives.extractUuid
+import org.genivi.sota.marshalling.RefinedMarshallingSupport._
+
+import scala.concurrent.ExecutionContext
+
+trait DeviceDirectives extends Directives {
```

Comment:
We should favour including `Directives` rather than extending it: `import Directives._`